packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -688,6 +688,9 @@ void main() {
       trackWidgetCreation: true,
     )).generator as DefaultResidentCompiler;
 
+    expect(residentCompiler.librariesSpec,
+      globals.fs.file(globals.artifacts.getArtifactPath(Artifact.flutterWebLibrariesJson, mode: BuildMode.debug))
```

Comment:
Why does this one pass a `mode`, but the call in resident_runner.dart doesnt?

Reply:
Good point, It shouldn't be necessary - removed